pkg/plugin/system/system_test.go

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright © 2022 Aspect Build Systems Inc
+Not licensed for re-use.
+*/
+
+package system
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	goplugin "github.com/hashicorp/go-plugin"
+	. "github.com/onsi/gomega"
+
+	"aspect.build/cli/pkg/ioutils"
+	"aspect.build/cli/pkg/plugin/sdk/v1alpha2/plugin"
+)
+
+// Finder returning an error
+type errorFinder struct{}
```

Comment:
Use gomock instead for this and the below structs.

Reply:
Done